source/include/ota.h

```diff
@@ -212,7 +204,7 @@ typedef void (* OtaAppCallback_t)( OtaJobEvent_t eEvent,
                                    const void * pData );
 
 /**
- * @ingroup ota_datatypes_functionpointers
+ * @ingroup ota_callback_types
```

Comment:
Missed periods in both the `@param`s. (L213-214)

Reply:
Done: https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/116/commits/3f19470223e963500c4fc69e1756bb21292903a9